versioneer.py

```diff
@@ -0,0 +1,1855 @@
+
+# Version: 0.19
+
+"""The Versioneer - like a rocketeer, but for versions.
+
+The Versioneer
+==============
+
+* like a rocketeer, but for versions!
+* https://github.com/python-versioneer/python-versioneer
+* Brian Warner
+* License: Public Domain
+* Compatible with: Python 3.6, 3.7, 3.8, 3.9 and pypy3
+* [![Latest Version][pypi-image]][pypi-url]
+* [![Build Status][travis-image]][travis-url]
+
```

Comment:
I this a file from `py-versioneer`? If so, are we allowed to bundle it with our code?

Reply:
As far as I understand how the versioneer works yes. I followed the install instructions under https://github.com/python-versioneer/python-versioneer/blob/master/INSTALL.md and this file got generated when installing the versioneer.

Reply:
I read quickly through it. As far as I understand `python-versioneer` creates these scripts here. Does it qualify as a dependency in the `pyproject.toml `? If I understand it correctly, we deliver the module `versioneer` via the `versioneer.py` so we do not rely on `python-versioneer` to be installed?!

Reply:
Correct. I removed the dependency in 2a28881 and uninstalled `versioneer` from my system. Everything is still working.